server/game/CardSelector.js

```diff
@@ -0,0 +1,141 @@
+const defaultProperties = {
```

Comment:
Can I be annoying and ask that these go in separate files?  In a folder together if necessary

Reply:
Not annoying, probably should have done it in the first place. It was convenient to have them all in one place during refactoring.